spec/models/good_spec.rb

```diff
@@ -4,4 +4,40 @@
   describe 'associations' do
     it { should belong_to(:basket) }
   end
+
+  describe 'tax calculation' do
+    before :each do
+      @importexampt = Good.new(name: 'imported chocolat', quantity: '1', price: '10.00')
+      @imported = Good.new(name: 'imported', quantity: '1', price: '47.50')
+      @exampt = Good.new(name: 'book', quantity: '1', price: '12.49')
+      @basic = Good.new(name: 'music', quantity: '1', price: '14.99')
+    end
```

Comment:
Here you can create factories or fixtures, and reuse them throughout your code. More on that here: https://semaphoreci.com/community/tutorials/working-effectively-with-data-factories-using-factorygirl